<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>803</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>-10</y>
      <width>591</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文新魏</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>企业打卡系统</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="CT_Img_Label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>400</width>
      <height>400</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文仿宋</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>人脸图像</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startPushButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>500</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文仿宋</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>开始识别</string>
    </property>
   </widget>
   <widget class="QTableView" name="basicTableView">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>560</y>
      <width>651</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>984</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
